tvb/recon/cli/mrtrix.py

```diff
@@ -165,11 +167,34 @@ class tck2connectome(BaseMtrixCLI):
     """
 
     class Flags(BaseMtrixCLI.Flags):
-        assignment_radial_search = "-assignment_radial_search"
-        assignment_end_voxels = "-assignment_end_voxels"
+        out_assignments = "-out_assignments"
         scale_length = "-scale_length"
         stat_edge = "-stat_edge"
 
+    class Assignment(metaclass=abc.ABCMeta):
```

Comment:
Let me know if you like this. It seems unnecessary verbose to me, but I couldn't think of a better way.